lib/fbe/middleware/sqlite_store.rb

```diff
@@ -16,11 +16,13 @@
 class Fbe::Middleware::SqliteStore
   attr_reader :path
 
-  def initialize(path)
+  def initialize(path, fbe_version)
```

Comment:
@Yegorov try to avoid compound/composite variable names: https://www.yegor256.com/2015/01/12/compound-name-is-code-smell.html